lxd/lifecycle/cluster_member.go

```diff
@@ -0,0 +1,35 @@
+package lifecycle
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/lxc/lxd/shared/api"
+)
+
+// ClusterMemberAction represents a lifecycle event action for cluster members.
+type ClusterMemberAction string
+
+// All supported lifecycle events for cluster members.
+const (
+	ClusterMemberDeleted      = ClusterMemberAction("deleted")
+	ClusterMemberUpdated      = ClusterMemberAction("updated")
+	ClusterMemberRenamed      = ClusterMemberAction("renamed")
+	ClusterMemberTokenCreated = ClusterMemberAction("token-created")
```

Comment:
I think this should be a Cluster event instead of a ClusterMember event as the token is used to join the cluster and therefore the member doesn't yet exist